frontend/public/components/dashboards-page/overview-dashboard/queries.ts

```diff
@@ -26,8 +26,8 @@ const overviewQueries = {
   [OverviewQuery.STORAGE_TOTAL]: 'sum(node_filesystem_size_bytes)',
   [OverviewQuery.PODS_BY_CPU]: 'sort_desc(pod_name:container_cpu_usage:sum)',
   [OverviewQuery.PODS_BY_MEMORY]: 'sort_desc(pod_name:container_memory_usage_bytes:sum)',
-  [OverviewQuery.PODS_BY_STORAGE]: 'sort_desc(avg by (pod_name)(irate(container_fs_io_time_seconds_total{container_name="POD", pod_name!=""}[1m])))',
-  [OverviewQuery.PODS_BY_NETWORK]: `sort_desc(sum by (pod_name)(irate(container_network_receive_bytes_total{container_name="POD", pod_name!=""}[1m]) + 
+  [OverviewQuery.PODS_BY_STORAGE]: 'sort_desc(avg by (pod_name, namespace)(irate(container_fs_io_time_seconds_total{container_name="POD", pod_name!=""}[1m])))',
+  [OverviewQuery.PODS_BY_NETWORK]: `sort_desc(sum by (pod_name, namespace)(irate(container_network_receive_bytes_total{container_name="POD", pod_name!=""}[1m]) + 
```

Comment:
Nit: no need for backticks unless there are expressions.

Reply:
we need them because the string is 2 lines long. I could use apostrophe, but then I'd have to move the query to one line which would be very long

Reply:
fixed